packages/jaeger-ui/src/model/transform-trace-data.tsx

```diff
@@ -159,6 +159,22 @@ export default function transformTraceData(data: TraceData & { spans: SpanData[]
   const tracePageTitle = getTracePageTitle(spans);
   const traceEmoji = getTraceEmoji(spans);
   const services = Object.keys(svcCounts).map(name => ({ name, numberOfSpans: svcCounts[name] }));
+
+  // Detect orphan spans - spans that reference a parent span ID that doesn't exist in the trace
+  let orphanSpanCount = 0;
+  spans.forEach(span => {
+    if (Array.isArray(span.references) && span.references.length > 0) {
+      const parentRef = span.references[0];
+      if (
+        (parentRef.refType === 'CHILD_OF' || parentRef.refType === 'FOLLOWS_FROM') &&
```

Comment:
this condition redundant because it is always true, since these are the only two valid ref types